cpp/src/ray/agent.cc

```diff
@@ -0,0 +1,17 @@
+#include <dlfcn.h>
+#include <stdint.h>
+
+namespace ray {
+namespace api {
+
+/* tmp impl, mast define in cc source file of dylib */
```

Comment:
Can you elaborate on this comment? I don't understand what it means.
Also, what is an agent?

Reply:
Agent provide a fixed function which help to get base address. The function offset will be calculate by this address. Maybe I can rename agent to AddressHelper?